pom.xml

```diff
@@ -76,7 +76,7 @@
         <version.log4j>1.2.16</version.log4j>
         <version.log4j-extras>1.0</version.log4j-extras>
         <version.lucene>7.5.0</version.lucene>
-        <version.microservice.accumulo-utils>1.2</version.microservice.accumulo-utils>
+        <version.microservice.accumulo-utils>1.3-SNAPSHOT</version.microservice.accumulo-utils>
         <version.microservice.audit-api>1.4</version.microservice.audit-api>
```

Comment:
this has to be 1.3 (not SNAPSHOT) once the accumulo-utils module is 'released' at 1.3